### PR TITLE
add validation to omit empty string from cardholder information

### PR DESCRIPTION
**Pull request by adhiartaputra**
- Add validation to remove empty string for cardholder information on token creation

### CODE REVIEW

**Review by azaky — APPROVED** [review]

File: Sources/Xendit/Models/DTOs/XenditTokenizationRequest.swift [hunk 1]
```diff
@@ -61,6 +61,11 @@ import Foundation
             "card_holder_phone_number": cardData.cardHolderPhoneNumber ?? ""
         ]
         
+        if cardData.cardHolderFirstName != nil && cardData.cardHolderFirstName != "" { cardDataJson["card_holder_first_name"] = cardData.cardHolderFirstName }
+        if cardData.cardHolderLastName != nil && cardData.cardHolderLastName != ""  { cardDataJson["card_holder_last_name"] = cardData.cardHolderLastName }
+        if cardData.cardHolderEmail != nil && cardData.cardHolderEmail != "" { cardDataJson["card_holder_email"] = cardData.cardHolderEmail }
+        if cardData.cardHolderPhoneNumber != nil && cardData.cardHolderPhoneNumber != "" { cardDataJson["card_holder_mobile_number"] = cardData.cardHolderPhoneNumber }
+
```
Comment (pandyaka) [inline Sources/Xendit/Models/DTOs/XenditTokenizationRequest.swift:68]: nit:

```
if let firstName = cardData.cardHolderFirstName, !firstName.isEmpty {
    cardDataJson["card_holder_first_name"] = firstName
}

if let lastName = cardData.cardHolderLastName, !lastName.isEmpty {
    cardDataJson["card_holder_last_name"] = lastName
}

if let email = cardData.cardHolderEmail, !email.isEmpty {
    cardDataJson["card_holder_email"] = email
}

if let phoneNumber = cardData.cardHolderPhoneNumber, !phoneNumber.isEmpty {
    cardDataJson["card_holder_mobile_number"] = phoneNumber
}
```

Comment (adhiartaputra) [reply Sources/Xendit/Models/DTOs/XenditTokenizationRequest.swift:68]: updated

File: Sources/Xendit/Models/DTOs/XenditTokenizationRequest.swift [hunk 2]
```diff
@@ -61,6 +61,11 @@ import Foundation
             "card_holder_phone_number": cardData.cardHolderPhoneNumber ?? ""
```
Comment (azaky) [inline Sources/Xendit/Models/DTOs/XenditTokenizationRequest.swift:61]: sorry I missed this before, as it's not part of the changes. should this be removed in the first place?

**Review by pandyaka — APPROVED** [review]

File: Sources/Xendit/Models/DTOs/XenditTokenizationRequest.swift (hunk 2 above)
Comment (adhiartaputra) [reply Sources/Xendit/Models/DTOs/XenditTokenizationRequest.swift]: @azaky please check a new PR, we deleted it

**Review by azaky — APPROVED** [review]

**Review by pandyaka — APPROVED** [review]

**Pull request merged**